client/src/api/resources/projects.js

```diff
@@ -87,6 +89,7 @@ export default {
 
   featured(limit = 6, filterParams = {}) {
     const filter = filterParams;
+    filter.draft = false;
```

Comment:
This is incorrect as well. If a project is a draft project, it's never visible to a non-admin user, regardless of whether or not it's featured. See why we need to handle this check server side?